meta/main.yml

```diff
@@ -12,4 +12,5 @@ galaxy_info:
   #   versions:
   #   - 2013.09
   galaxy_tags: [amazon, centos6, nginx, php, laravel]
-dependencies: []
+dependencies:
+  - { role: retr0h.logrotate, when: laravel_rotate_nginx_log }
```

Comment:
`{ role: retr0h.logrotate,2.0, when: laravel_rotate_nginx_log }`

- should pin it to the `role` 2.0 version.